code/src/index.css

```diff
@@ -1,13 +1,72 @@
+  
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #000000;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.movie-card-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.movie-card {
+  width: 100%;
+  position: relative;
+}
+
+.poster-image {
+  width: 100%;
+}
+
+.movie-details {
+  display: flex;
+  position: absolute;
+  flex-direction: column;
+  justify-content: flex-end;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0,0,0.6);
+  opacity: 0;
+}
+
+.movie-details:hover {
+  opacity: 1;
+}
+
+.main-title {
+  color: white;
+  padding-left: 8%;
+  font-size: 2.5vw;
+}
+
+.released-info {
+  color: white;
+  margin-top: 0;
+  padding: 0 0 5% 8%;
+  font-size: 2vw;
+}
+
+@media (min-width: 767px) {
+
+.movie-card {
+  width: 50%;
+}
+}
+
+@media (min-width: 1200px) {
+
+ .movie-card {
+  width: 25%;
+}
```

Comment:
This part of the code we consider it to be nested and therefore it needs some indentation, in contrast to all the code above. Two spaces indentation according to Technigo guidelines. 🍻